pkg/strategy/controller.go

```diff
@@ -184,6 +186,39 @@ func disableDependenciesTab(uiOpts map[string]interface{}, storage string, depsE
 	}
 }
 
+func enableLogOut(uiOpts map[string]interface{}, spec *v1.JaegerSpec) {
+	if (spec.Ingress.Enabled != nil && *spec.Ingress.Enabled == false) ||
+		spec.Ingress.Security != v1.IngressSecurityOAuthProxy {
+		return
+	}
+
+	if _, ok := uiOpts["menu"]; ok {
+		return
+	}
+
+	menuStr := `[
+		{
+		  "label": "About",
+		  "items": [
+			{
+			  "label": "Documentation",
+			  "url": "https://www.jaegertracing.io/docs/latest"
+			}
+		  ]
+		},
+		{
+		  "label": "Log Out",
```

Comment:
This would render the "Log out" right next to the "About" item, right? Are they aligned to the main options, or are they aligned to the right?

Reply:
To the right.

Reply:
Great! Looks good to me, then